source/quick-start.txt

```diff
@@ -39,32 +82,21 @@ Use the ``dotnet add`` command to add the {+driver-short+} to your project as a
 
    dotnet add package MongoDB.Driver
 
-Create a MongoDB Cluster
-------------------------
-
-.. include:: /includes/quick-start/atlas-setup.rst
-
 Query Your MongoDB Cluster from Your Application
 ------------------------------------------------
 
-First, save your Atlas connection string to an environment variable named ``MONGODB_URI``. This prevents potential security issues caused by including your credentials in the source code.
+In this step, you'll use the {+driver-short+} 
+to connect to your MongoDB cluster and run a query on the sample data.
 
-.. code-block:: bash
-
-   export MONGODB_URI='<your atlas connection string>'
-
-.. important::
-
-   Make sure to replace the "<username>" and "<password>" sections of the connection string with the username and password of your Atlas user.
```

Comment:
Suggestion: Could we add a version of this admonition to where we tell the user to create their environment variable? I think it's still good to point this out in case some users are just copy/pasting and don't realize the placeholders are there.